scripts/globals.py

```diff
@@ -2,26 +2,54 @@
 import os
 
 import modules.scripts as scripts
-import modules.shared as shared
+from modules.shared import opts, models_path, cmd_opts
+from modules.cmd_args import parser
 
 
-global hide_nsfw_networks_key, hide_nsfw, network_descriptions_path
+global hide_nsfw
 
 
-hide_nsfw_networks_key = "sd_lora_tagger_hide_nsfw_extra_networks"
-network_descriptions_path = os.path.join(scripts.basedir(), "network_descriptions")
+def out(msg):
+    print(f"SD Lora Tagger: {msg}")
+
+
+""" Paths """
+lora_tagger_dir = scripts.basedir()
+models_dir = models_path
+network_descriptions_path = os.path.join(lora_tagger_dir, "network_descriptions")
 
-print(f"SD Lora Tagger: network_descriptions_path={network_descriptions_path}")
+# Used in accordance to https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/master/modules/cmd_args.py
+model_dirs = {
+    "Lora": cmd_opts.lora_dir,
+    "LyCORIS": os.path.join(models_dir, 'LyCORIS'),
```

Comment:
Leftover from my patch (4ec8b9e). LyCORIS path still needs to be implemented in a more dynamic way. If I'm not mistaken, the LyCORIS extension adds it's own command-line argument similar to what I've used.

Reply:
Based on what I've found, loras and lycos use the same directory scheme, so the lyco directory is just cmd_opts.lyco_dir for our intents and purposes.

```
opts.onchange("lora_dir", lambda: setattr(args, "lora_dir", opts.lora_dir))
opts.onchange("lyco_dir", lambda: setattr(args, "lyco_dir", opts.lyco_dir))
```